### ISSUE TITLE
Are Statements fit for purpose?

### ISSUE THREAD

**Issue by mnot** [w3c/w3process#712]
Some folks say they're planning/hoping/assuming that documents (e.g., Ethical Web Principles, Vision) which don't come from the WG consensus process might become [Statements](https://www.w3.org/2021/Process-20211102/#memo), and thus represent W3C community consensus.

This makes me uncomfortable. Until now, W3C community consensus has been acheived by allowing any member of the community to participate in a process that is well-defined, with many guiderails to assure that voices are hard, power is not abused, etc. In a WG, I know that I can say my piece, and if I convince my peers, the chair (who is neutral) will instruct the editors to reflect consensus in the document. I know that if I disgree with a decision, there is a well-defined appeal pathway. I know that decisions can't be made without proper notice and transparency. And so forth.

A non-WG forum (for example, the AB or TAG) doesn't have these properties. I am aware and appreciative of efforts to be inclusive, listen to outsiders, consider all input, etc. However, they are not enough; decisions are left in the hands of those who run the body, and any 'consensus' determination they make cannot be challenged. It is not a community decision; it is a decision of that body's members, no matter how hard they strive to be open. 

It is true that a Statement needs to be ratified by the AC first. However, this can't be equated to the ratification step in normal Recommendation processing, because it presents an all-or-nothing proposition to members; they had no formal opportunity to participate in the development of the document and negotiate consensus (with the protections discussed above) in the group; only an ex post up-or-down vote in the AC. Yes, of course they can informally participate in the open discussions of the documents being formed up as Statements; however, the fact is that the power structure in place in these non-WG fora _fundamentally_ changes how people participate in them (or decide not to).

Compounding this is the limits on who can appeal the promotion of a Statement -- only an AC rep can. A member of the public or someone who can't convince their AC rep to make waves is out of luck. Thus, the overall community thus gets no role in approving what is supposed to represent it; only the AC (whose dysfunction is well-recognised).

In short, Statements feel like a shortcut -- it's too onerous to get community consensus in a WG, so we'll delegate the consensus process to a smaller group, and then give the community an (unlikely to be excercised) opportunity to reject the result. That doesn't seem like real consensus, especially when we're talking about fundamental issues like values and principles.

Understand -- this is not a complaint about how those groups are running their processes per se; they appear to have put monumental efforts into soliciting feedback, review, and input, and appear to be listenting intently.

**Comment by chaals** [comment]
Yeah, I have a similar sense of unease about the "statement" path.

**Comment by dwsinger** [comment]
Hi Mark, I think you may be under a misapprehension. The statement track is not exclusive to the AB and TAG: it is a track available to any chartered group to produce a W3C consensus document that is not a normative specification.

Now, the AB and TAG are formally restricted from producing Recommendations, because participation is not linked to the Patent Policy (I know, there is a record of the TAG having made one). However, when designing the process for Statements, there did not seem to be a strong reason (then) to restrict the TAG and AB from asking to elevate Notes into Statements. So the track is available to them. You seem to be arguing that perhaps it should not be. 

So, it's not that Statements are a shortcut, I think, it's that Statements from elected groups such as the AB and TAG might be less appropriate. We tend to treat the EWP and WPDP principles as having some status, but they don't; I would like them to have the status we need and they deserve, and I don't agree it's a demerit that they were authored by the TAG.

There's another misapprehension here, I think. Recommendations and Statements are required to have had wide review, ideally public review and certainly member review. A valid ground for complaint, or even formal objection, from anyone, would be that there has not been wide review or that its results have been ignored. So I don't think "getting to your AC Rep" is your only avenue of recourse, either.

Finally, it's not that making a WG is too onerous; it's that we had no way to agree on anything other than normative specifications, Recommendations. I personally doubt we'd be successful launching a working group to refine (e.g.) the Ethical Web Principles; I think that people feel that this kind of shepherding is exactly why we elect the TAG and AB. (And the process CG is very lightly attended, so not an encouraging example.) 

But I do agree that we need to do more than the 'here is the document and its repo, feel free to raise issues or submit pull requests'. For a start, without an incentive or deadline, people rarely pay attention. Secondly, some things really are best discussed person to person, in real time. So I agree having focused meetings, breakouts, on such documents, would be important.

**Comment by mnot** [comment]
> Hi Mark, I think you may be under a misapprehension. The statement track is not exclusive to the AB and TAG: it is a track available to any chartered group to produce a W3C consensus document that is not a normative specification.

No, I didn't assume that; it's just what's most talked about now.

> So the track is available to them. You seem to be arguing that perhaps it should not be.

I'm expressing concern that Statements seem to be equated by some with having community consensus, and pointing out how they may not be suitable for serving that purpose, when used by bodies like the TAG and AB.

> So, it's not that Statements are a shortcut, I think, it's that Statements from elected groups such as the AB and TAG might be less appropriate.

Sure -- let's say it's possibly a shortcut *for them*. Although I struggle to imagine another group that isn't going to have the same set of potential problems, if it's not a WG.

> We tend to treat the EWP and WPDP principles as having some status, but they don't; I would like them to have the status we need and they deserve,

Agreed

>  and I don't agree it's a demerit that they were authored by the TAG.

That's your misapprehension, not mine.

> Recommendations and Statements are required to have had wide review, ideally public review and certainly member review. 

Who judges whether the review is wide, and how?

> A valid ground for complaint, or even formal objection, from anyone, would be that there has not been wide review or that its results have been ignored. So I don't think "getting to your AC Rep" is your only avenue of recourse, either.

Looking at the [new process](https://www.w3.org/Consortium/Process/Drafts/), this is far from clear. The [relevant text](https://www.w3.org/Consortium/Process/Drafts/#registering-objections) says:

```
Any individual (regardless of whether they are associated with a Member) may 
appeal any decision made in connection with this Process (except those having 
a different appeal process) by registering a Formal Objection with the Team.
```

Fine. However, when [talking about Statements, it says](https://www.w3.org/Consortium/Process/Drafts/#memo):

```
The decision to advance a document to W3C Statement is a W3C Decision.
Advisory Committee representatives may initiate an Advisory Committee 
Appeal of the decision.
```

That seems to establish a 'different appeal process', opting these decisions out of the opportunity for a public FO.

Even if an FO were available, keep in mind that it would be handled by a council whose composition was half filled by the very people who originally approved the document if it was the AB or TAG, and the other half would have a presumably close working relationship with them (and perhaps some shared employers). The recusals would be fraught, and establishing believable legitimacy could be very difficult. This is hardly the hallmark of a consensus process.

> Finally, it's not that making a WG is too onerous; it's that we had no way to agree on anything other than normative specifications, Recommendations. I personally doubt we'd be successful launching a working group to refine (e.g.) the Ethical Web Principles; I think that people feel that this kind of shepherding is exactly why we elect the TAG and AB. (And the process CG is very lightly attended, so not an encouraging example.)

This very much begs the question of _why_ you think such a WG would not be successful. Indeed, given that some people won't be willing to put in the effort in a CG or other informal body when they know they can be ignored/sidelined, I'd think it might be the opposite. If WGs can't work on anything but normative specifications, why don't we make the minimal change and **fix that**?

> For a start, without an incentive or deadline, people rarely pay attention.

I'd add that people rarely invest time and attention when they arne't considered equal stakeholders in the process (no matter how much the actual leaders bend over to try to listen to them).

**Comment by mnot** [comment]
Also, I'll emphasise that there's a fundamental difference between a process where the safeguards / consensus-balancing mechanisms are operating _during_ the document's creation, and one where they're inserted afterwards in a ratifying step. 

This is true whether it's through a FO, an AC Review, or whatever other ex post process is used.

**Comment by frivoal** [comment]
> That seems to establish a 'different appeal process', opting these decisions out of the opportunity for a public FO.

It does not. The exact same thing is true for RECs: AC Review (during which Formal Objections, if any, will be handled), then the conclusion of the AC Review is a W3C Decision, which can be appealed.

> I'm expressing concern that Statements seem to be equated by some with having community consensus

A document which is being prepared to become a Statement but isn't yet a Statement (i.e., a Note) does not have consortium-wide consensus, just like a WD or a CR do not have consortium wide consensus. In order to get there, you will need to formally address all issues raised against the document since its publication as a Note (not just by group members, but by anybody, as would be the case of a REC track document), undergo wide review (which includes, but isn't limited to Horizontal Review), and to succesfully pass an AC Review. Patent Policy and software implementation aside, this is the same as for a REC.

> Who judges whether the review is wide, and how?

The exact same people who judge if for a REC (the Team), under the same criteria: have you followed the process for Horizontal Review, and have you solicited and recieved review from a broad range of people.

And if AC Reps are not satisfied with the Team's evaluation, Formal Objections can be filed. And if the council does a lousy job, an AC appeal can be filed too.

Not claiming it's perfect, but some you seem to be thinking that we can just work out a document in isolation, then submit it for ratification and be done. But that is not the Process, and doing that would/should result in failing to get to the Statement status.

**Comment by dwsinger** [comment]
I still think we don't understand each other; I am fairly sure I don't understand you. I think you might be saying that if an elected group (as opposed to a voluntary-participation group) wants to create a Note and then put it on the Statement track, they are going to have to make sure that the community had ample opportunity to participate and contribute, and that to be acceptable as a Statement, that participation and wide review are going to be important. I doubt anyone disagrees.

I think you might be saying that elected groups should not be able to propose their Notes as Statements, but then you seem to agree that the TAG's EWP and WPDP should be elevated, so I am not sure. I note that both the [IESG](https://datatracker.ietf.org/doc/search?csrfmiddlewaretoken=GSK7H2s3b4rtBccIj87ccHhXFTSMDUQM7FC34I9TXEQqnIWXyYoeaZHmVhI0e24J&name=&sort=&rfcs=on&activedrafts=on&by=group&group=iesg) and [IAB](https://datatracker.ietf.org/doc/search?csrfmiddlewaretoken=GSK7H2s3b4rtBccIj87ccHhXFTSMDUQM7FC34I9TXEQqnIWXyYoeaZHmVhI0e24J&name=&sort=&rfcs=on&activedrafts=on&by=group&group=iab) at the IETF author RFCs.

I have no doubt that the Vision document needs more community involvement and input (though, to the best of my knowledge, the TAG has maintained their documents on their own, without any explicit request for community input). I repeat, the AB does not believe that they 'own' the Vision, but that it's their job to catalyze its development _in community_. If the AB does not step up, I am confident that no-one else would; we'd be [expecting a miracle](https://www.pinterest.com/pin/and-then-a-miracle-occurs--238409374008153066/).

You say you understand that Statements are available to all groups, but then ask "If WGs can't work on anything but normative specifications, why don't we make the minimal change and fix that?". Statements do fix that; Statements can be proposed by any group, elected or chartered. So I continue to think there is misunderstanding here, as the "if" here doesn't hold.

I have a suspicion we may be in violent agreement; the W3C needs to be clearer about its vision and principles, that they need to be built and endorsed in community, and it's the job of the elected groups (AB, TAG, and to a small extent the Board) to make sure that happens. Which is what we are trying to do. I think that the AB recognizes that to go any further than the nucleus primarily developed by contribution from AB members (but not exclusively), we'll need to ramp up community engagement. How to do that is a key question; at the moment, while it remains in draft form (it's not even yet styled as a W3C note), continued exposure to the AC and members seems like one good step.  I think we might be in violent agreement about the EWP and WPDP also, that it's time to look at getting W3C Consensus, Statement status, on them as well (the lack of explicit community engagement being balanced by their age/maturity and apparent general acceptance, perhaps).

**Comment by TzviyaSiegman** [comment]
This is a very interesting discussion. If I'm understanding @mnot correctly, his position is that anything that is statement (or rec) track, should be developed in a working group to facilitate broader participation in the development (not just review) of these documents. 

I think this is a legitimate position that should trigger us to think about how tightly scoped WGs are today. On the flip side, I do think that we need to leave room for TAG and AB to publish. I am wondering what the right mechanism is to encourage contributions, create an inclusive environment, and still have documents that are driven and opinionated. Indeed, one of the goals of EWP is to make sure that those who don't agree with the principles are not involved. Maybe a discussion for Gov week?

**Comment by chaals** [comment]
> ... I do think that we need to leave room for TAG and AB to publish.

We explicitly ask them to publish certain things (findings, W3C process) and to provide input to other groups and their work.

I'm leaning toward the idea that if they want to publish something as a consensus of the W3C, they should develop it through the existing processes everyone else uses. There's nothing to stop them proposing a Working Group - and I believe (this is 20-odd years ago so my memories might not match what others remember) that's basically how the Patent Policy was created.

> I am wondering what the right mechanism is to encourage contributions, create an inclusive environment, and still have documents that are driven and opinionated. 

We usually do that with Working Groups, and I think @mnot's suggestion is that this is sufficient and it is helpful not to multiply the pathways.

> Indeed, one of the goals of EWP is to make sure that those who don't agree with the principles are not involved.

Oh. I hope that isn't the case - I think it's important that we can be challenged by people who disagree with us. I think the point of having consensus as the basis on which we aim to decide is important. If the EWP is intended to qualify whose opinion counts in a consensus, I'm concerned that could distort our governance by silencing future discussions of our values.

If I "counter-propose" that the goal of EWP (and other things like it) is to make it clear that there are principles underpinning our approach and unless we are convinced that the principle is wrong we are unlikely to agree with arguments that counter those principles, do we agree on the goal?

> Maybe a discussion for Gov week?

Feels like a discussion for the AC, to me.

**Comment by dwsinger** [comment]
I think the result of a policy that said that if an elected group (AB, TAG) wanted to get W3C consensus on something, they'd have to write a charter, get charter approval and a working group formed, get enough people in it, and then go from working draft in the WG all through the process of document development, would be that we'd end up with nothing.

I'm fine with a (valid) objection to elevating something to consensus status being that it's not had enough community input, review, or that the results of that input and review have not been duly considered. That's fair and normal for any publication. 

But also: imagine the TAG did create a WG to carry EWP to statement, and one to carry WPDP to statement : and that the WG was lightly attended, dominated by people from the TAG. Either we'd be using the WG 'label' on that work as a screen/excuse, or we'd be back where we started.

I doubt that the overall problem is whether the community input has occurred _in a WG_. The problem is whether it's happened and been taken into account at all. "You could have joined the WG and didn't" is not something we can say at AC review today about Recommendations and should not be something we could imagine saying about Statements.

Let's get the community involvement addressed, for all documents no matter their shepherding group.

**Comment by frivoal** [comment]
> I'm leaning toward the idea that if they want to publish something as a consensus of the W3C, they should develop it through the existing processes everyone else uses.

Yes, which is to publish a document, get wide/horiztonal review, receive feedback, address it, iterate until satisfaction, get an AC Review… and to have their work rightfully rejected if they didn't go a good enough job, or endorsed if they did.

A REC or Statement does not document the mere consensus of the participants of the Working Group, but of the whole consortium, thanks to the various requirements for wide/horizontal/AC review, formally addressing issues… Yes, the work is lead by that group, but to go beyond draft stage, you'll need more than just the Group's internal consensus.

Just the same, an AB or TAG document should get to be a consortium-endorsed statement not because we expect the whole consortium to join the AB or TAG (or a group that they would spawn) any more than we expect the whole consortium to join a Working Group, but because we'd expect them to do a good enough job of subjecting their work to review and addressing it.

To me, claiming that the AB / TAG cannot produce documents getting the approval of the whole consortium because not everybody can join the AB / TAG means that the whole wide review / horizontal review / AC Review is considered to add little value, and that RECs only really carry the endorsement of their originating group and not really much more. That would be a very sad conclusion.

**Comment by michaelchampion** [comment]
Agree with @frivoal but one friendly amendment:  The AC should have an explicit invitation to review works BEFORE they are locked down and the only option is some variation on Approve,  or Formal Objection.  This seems to be current practice, but it is not AFAIK in the Process. 

There is a cultural norm in W3C against formal objection unless "you think the Recommendation/Charter would drive the web off a cliff" as some team member put it back when I was an AC Rep. This leads to a strong bias in favor of approving whatever the Team (or in this case the TAG or AB) puts forward for approval.   The cumbersome Formal Objection Council mechanism may perpetuate this sense that objecting in an AC ballot is a "nuclear option".  

Since I'm talking about practice and culture not Process here, my friendly amendment is to urge the TAG/AB to run a call for review / consensus with the AC before submitting for formal approval documents like the Ethical Web Principles and Vision.  That is, point to the draft document and ask for -1 / 0 / +1 on whether to advance it.  Then work with -1 voters to determine what changes -- if any -- would cause them to not object in the formal vote, then decide whether to make them or not.

**Comment by frivoal** [comment]
> The AC should have an explicit invitation to review works BEFORE they are locked down and the only option is some variation on Approve, or Formal Objection. This seems to be current practice, but it is not AFAIK in the Process.

I agree. The Process does lean that way, but falls short of being fully explicit about it. The WG/AB/TAG is required to solicit wide review, and to respond to any comment made. Wide review means making sure you've engaged with relevant audiences/stakeholders, so that you didn't cook the document alone in your corner without getting relevant feedback. Especially for this kind of document, if you haven't reached out to the AC, it'd be hard to argue that you did wide review properly, even if the Process doesn't spell it out.

**Comment by nigelmegitt** [comment]
> if you haven't reached out to the AC, it'd be hard to argue that you did wide review properly,

I generally consider reaching out to specific subsections of W3C membership to be "narrow" review and explicitly _not_ wide review. It's easy to publish a document, send some comms to external (e.g. liaison) organisations, tweet about it, publish a W3C blog post, and claim you've solicited wide review, then process any feedback arising, to complete the wide review process, without ever _explicitly_ consulting AC - after all, AC consists of people who had the same opportunity to see the wide review call and feed back as everyone else, so by definition they were "included".

If an AC review is important (which is arguably is for Statements), it should be called out as a requirement.

**Comment by mnot** [comment]
Sorry, it's been a busy week and I'm recovering from COVID, so my capacity is limited. A few responses to comments since my last:

@frivoal:

>> That seems to establish a 'different appeal process', opting these decisions out of the opportunity for a public FO.
>
> It does not. The exact same thing is true for RECs: AC Review (during which Formal Objections, if any, will be handled), then the conclusion of the AC Review is a W3C Decision, which can be appealed.

See my comment just before yours. There's a big difference between a document being developed in a group where one has a formal role, means of appeal, oversight mechanisms, etc. vs. one that doesn't. Ratification is simply that; it's an up-or-down vote on whether the product should be approved, and doesn't afford the same opportunity to participate in the document's development.

To give a concrete example, the Vision document is being developed by the AB. They are doing everything they can to solicit others to contribute, and that's great. However, the _evaluation_ of those contributions is entirely up to them; they can selectively ignore or minimise outside views at their pleasure. It is a document that reflects AB consensus; it _cannot_ represent any wider consensus. Taking it to the AC for ratification as something repersenting community consensus is inappropriate, because that is an insufficient mechanism on its own to judge true community consensus.

> And if AC Reps are not satisfied with the Team's evaluation, Formal Objections can be filed. And if the council does a lousy job, an AC appeal can be filed too.

See my previous comment about how the Council is loaded with people who worked on these documents, or have close relationships with them. And how the AC is a poor representation of the community.

@dwsinger:

> I think you might be saying that elected groups should not be able to propose their Notes as Statements, but then you seem to agree that the TAG's EWP and WPDP should be elevated, so I am not sure. 

No, apologies if there was any confusion; I think all these documents suffer from these concerns, if they're following that path.

> I note that both the [IESG](https://datatracker.ietf.org/doc/search?csrfmiddlewaretoken=GSK7H2s3b4rtBccIj87ccHhXFTSMDUQM7FC34I9TXEQqnIWXyYoeaZHmVhI0e24J&name=&sort=&rfcs=on&activedrafts=on&by=group&group=iesg) and [IAB](https://datatracker.ietf.org/doc/search?csrfmiddlewaretoken=GSK7H2s3b4rtBccIj87ccHhXFTSMDUQM7FC34I9TXEQqnIWXyYoeaZHmVhI0e24J&name=&sort=&rfcs=on&activedrafts=on&by=group&group=iab) at the IETF author RFCs.

Those documents do not reflect community consensus; the IAB cannot and will not publish a document saying "IETF consensus is _this_." They represent those bodies in their defined roles, not the whole community.

> I have a suspicion we may be in violent agreement; the W3C needs to be clearer about its vision and principles, that they need to be built and endorsed in community, and it's the job of the elected groups (AB, TAG, and to a small extent the Board) to make sure that happens. Which is what we are trying to do. I think that the AB recognizes that to go any further than the nucleus primarily developed by contribution from AB members (but not exclusively), we'll need to ramp up community engagement. How to do that is a key question

We are entirely in agreement up to this point -- except, perhaps that _it's the job of the elected groups to make sure this happens._ I think it's the job of the whole community, with the Process as a regulator for that effort. 

Putting the responsibility as well as the judgement call on the leadership just leads us back to the old paternalistic ways of "don't worry, this is being taken care of for you." No wonder there's an engagement problem -- if people's agency is limited, and they're told that someone else owns something (or assume it because of the structure of the organisation), they won't be inclined to make an effort in all but the most extreme of circumstances.

Later:

> I doubt that the overall problem is whether the community input has occurred in a WG. The problem is whether it's happened and been taken into account at all.

Yes, but how you judge whether that's happened is critical. Right now the only mechanism that makes that judgement accountable is the distant threat of not being re-elected. The AC (as discussed elsewhere) is a very poor counterbalance for this power.

@TzviyaSiegman:

> Indeed, one of the goals of EWP is to make sure that those who don't agree with the principles are not involved.

To add to Charles' comments -- we have much bigger problems if this is true; I sincerely hope it is not. Happy to chat more about this offline with anyone who thinks it is (or should be) true.

**Generally**: I'm happy to see people talking about getting broader and more representative review of these documents, and agree that they don't strictly need to be developed in WGs. 

However, we need to look at the properties that WGs provide to consensus-gathering more carefully, e.g.: 

* A judge of consensus who's accountable (i.e., they can be removed)
* An appeal process for individual decisions, not just an up-or-down on the whole document
* Process for determining consensus, as well as things like notice periods, review stages, transparency, etc. etc.
* Broad ability to FO (not just the AC)

In the end, it might be easier to work in a WG than introduce these properties into TAG and AB documents. I also suspect it's the best way to address engagement problems -- if people have agency they're more likely to put in effort.

**Comment by dwsinger** [comment]
Thanks Mark, responses to some points inline

> To give a concrete example, the Vision document is being developed by the AB. They are doing everything they can to solicit others to contribute, and that's great. However, the _evaluation_ of those contributions is entirely up to them; they can selectively ignore or minimise outside views at their pleasure. It is a document that reflects AB consensus; it _cannot_ represent any wider consensus. Taking it to the AC for ratification as something repersenting community consensus is inappropriate, because that is an insufficient mechanism on its own to judge true community consensus.

I think this misses something quite fundamental. Wide review at the W3C _fails_ if the comments received in development were not taken into account, and the team should and the AC can and should point out when due consideration was not given to input received, and not approve progression. We even expect a disposition of comments report (e.g. as is done for the Process). We _have_ checks for this, and they have been seen to work.

> See my previous comment about how the Council is loaded with people who worked on these documents, or have close relationships with them. And how the AC is a poor representation of the community.

If the AC vote to progress fails, we don't even get to formal objections.

> 
> We are entirely in agreement up to this point -- except, perhaps that _it's the job of the elected groups to make sure this happens._ I think it's the job of the whole community, with the Process as a regulator for that effort.

I think this is a failure mode. Something that is the vague 'community's job is everyone's job, and we end up that no-one does it. There has to be leadership, catalysis, and I believe that the elected bodies are the right ones to provide focus, leadership, editing resources, and so on.

> Putting the responsibility as well as the judgement call on the leadership just leads us back to the old paternalistic ways of "don't worry, this is being taken care of for you." 

That is _not_ what anyone is saying. Quite the opposite: It's no longer "The Director embodies our principles, we don't need to worry", it's precisely that we need to discuss our principles and values explicitly, and yes, we elect groups to drive that kind of thing.

> No wonder there's an engagement problem -- if people's agency is limited, and they're told that someone else owns something (or assume it because of the structure of the organisation), they won't be inclined to make an effort in all but the most extreme of circumstances.

People have unlimited agency in the Process – the CG is open to anyone. How many do engage? I don't think that's why we have limited engagement. 

Mind you, I _agree_ and _support_ that we should find ways to enable the community to have full engagement, whatever that is. Maybe, despite the process example, a CG would help. Maybe a workshop. Maybe a limited set of meetings where people can debate in real time (the Github repo is already open to async debate).

I suspect we have limited engagement also because people are busy, or that they actually trust what's going on ("they seem to be doing a perfectly competent job without me", a common problem of standards engagement), or that they don't see how it will impact them. Overall it's easy for people to end up at "it's important this is done but I don't need to be involved", which is fine as long as they don't object later. 

What's not fine would be springing something on them and telling them that their opportunity to get involved was documented ["in a filing cabinet behind a door marked 'Beware of the leopard'"](https://www.goodreads.com/quotes/40705-but-the-plans-were-on-display-on-display-i-eventually). Or that the document was developed by a select group who neither sought nor accepted feedback and input.

> Yes, but how you judge whether that's happened is critical. Right now the only mechanism that makes that judgement accountable is the distant threat of not being re-elected. The AC (as discussed elsewhere) is a very poor counterbalance for this power.

No, the AC has to approve document progression. If the AC feels it's wrong and has not taken the correct perspective or their views into account, they can (and should) vote No. That means the TAG and AB have wasted their time. This is much more immediate than election. It's why we wanted a statement track; so that documents that sort-of have community status but sort-of don't (like TAG findings), get that clarified.

> **Generally**: I'm happy to see people talking about getting broader and more representative review of these documents, and agree that they don't strictly need to be developed in WGs.

Great, we are in violent agreement, and I think this is the key question: how do we get quality inspection, comment, and improvement on all these documents from as broad a set of the community as possible? I don't think we have the answer yet. 

But I am sure that saying that a mechanism that says that the TAG and AB should not lead, should not catalyze, and the community should develop them, is not a recipe for success. They should.

> In the end, it might be easier to work in a WG than introduce these properties into TAG and AB documents. I also suspect it's the best way to address engagement problems -- if people have agency they're more likely to put in effort.

I would be delighted if there were a scad of complaints to the AB and TAG that people want to be more involved and they find the existing mechanisms inadequate, because they might then be able to suggest better mechanisms…

**Comment by cwilso** [comment]
I would point out that the description of Statements does not ever say they represent consensus of the entire W3C Membership - but rather that a Statement has "been endorsed by W3C as a whole".  (Including addressing FOs, etc.- but does not claim it has been produced in an entirely open forum.)

It's already been pointed out that the TAG's Statement-track works are not produced in a more open way - I would further point out that given the stronger management of that effort (e.g. closing of conversations by the Chairs, or the Privacy Principles's closed-invite-only-membership group), they're even more directly "managed".  I'm not convinced that's a bad idea; they still need to be endorsed, of course, by the whole membership, but we do (as David says) need leadership.

**Comment by frivoal** [comment]
I have just proposed a potential charter for a task force to work on the Vision in https://github.com/w3c/AB-public/issues/57#issuecomment-1454378201, largely derived from the way the Process CG operates. I am mentioning it here, despite this issue being more general in purpose, because I think this is a way in which the AB could structure its work in a way that would be conducive to community-wide engagement, and to successfully passing the wide-review expectation. Not claiming it's the only way to go about it, but I suspect something along these lines would be a good way for groups like the AB or TAG to work on documents that are meant to become Statements.

**Comment by mnot** [comment]
@dwsinger:

> I think this misses something quite fundamental. Wide review at the W3C fails if the comments received in development were not taken into account, and the team should and the AC can and should point out when due consideration was not given to input received, and not approve progression. 

I understand that you believe the Team + AC review are suitable counterbalances for the power excercised by the AB and TAG in making Statements that purport to represent the whole community. I have been consistently arguing that they are not -- I'm not _missing_ anything, I'm disagreeing with that judgement call.

I've already talked extensively about how the Team's accountability is limited, and about the dysfunction of the AC as a review body or as a representative of the community. While they may be adequate counterbalances when the process that produced the document had robust protections, they fail when those protections are not present. 

Instead, there are a set of vague, objective goals ("broad review") that are opaquely judged. Expecting the AC to self-organise to enforce those goals means that only the most egregious violations have a hope of being called out. Anyone who attempts that knows that they're going to face inter-corporate politics, as well as the potential for things escaping into the media (as we've recently seen), despite the supposed Member-confidentiality of procedings. 

> We even expect a disposition of comments report (e.g. as is done for the Process). We have checks for this, and they have been seen to work.

That is by nature _ex post_; it doesn't help at the time there is a disagreement. By nature, the Process says that these documents contain whatever their sponsoring bodies want them to; there is no Process basis for complaining. I've raised this factor multiple times above, but haven't yet seen anyone engage with it.

Furthermore, will that report really reflect all of the disagreements with the document's contents? Will it reflect this discussion, for example, if it's not closed to my satisfaction? What has to happen for something to get into the report, and how are participants to know what that mechanism is?

> I suspect we have limited engagement also because people are busy, or that they actually trust what's going on ("they seem to be doing a perfectly competent job without me"

I agree those are likely factors. I'm encouraged by your "also" there.

> But I am sure that saying that a mechanism that says that the TAG and AB should not lead, should not catalyze, and the community should develop them, is not a recipe for success. They should.

I'm not proposing that, so no worries.

> I would be delighted if there were a scad of complaints to the AB and TAG that people want to be more involved and they find the existing mechanisms inadequate, because they might then be able to suggest better mechanisms…

What do you think this issue _is_?

@cwilso:

> I would point out that the description of Statements does not ever say they represent consensus of the entire W3C Membership - but rather that a Statement has "been endorsed by W3C as a whole". 

That's a _very_ fine hair to split, and I'm a bit surprised to see you try. People regularly refer to Statements as having "[consensus status](https://github.com/w3c/AB-memberonly/issues/150#issuecomment-1397506598)", and I think we'd have a really hard time explaining them as anything else externally.

@frivoal:

> I have just proposed a potential charter

Thanks; having a meaningful decision policy is a signficant step forward. I'm a bit nervous about the 'Explicit decisions of the AB are binding onto this group.', but if they're transparent, they can be evaluated reasonably.

**Comment by cwilso** [comment]
I just wanted to note for those watching this issue that we intend to have a breakout session on this topic at the upcoming AC meeting: https://www.w3.org/2023/05/AC/ac-agenda.html.
